frontend/src/model/beta/AccessRequests.tsx

```diff
@@ -19,9 +19,11 @@ export default function AccessRequests({ model }: AccessRequestsProps) {
   const accessRequestsList = useMemo(
     () =>
       accessRequests.length ? (
-        accessRequests.map((accessRequest) => (
-          <AccessRequestDisplay accessRequest={accessRequest} key={accessRequest.metadata.overview.name} />
-        ))
+        accessRequests
+          .sort((a, b) => b.createdAt.localeCompare(a.createdAt))
```

Comment:
This sort can be replaced by the reusable `sortByCreatedAtDescending` from `dateUtils.ts`